packages/foreman/rubygem-validates_lengths_from_database/rubygem-validates_lengths_from_database.spec

```diff
@@ -55,26 +47,27 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
+%exclude %{gem_instdir}/Appraisals
+%license %{gem_instdir}/LICENSE
 %{gem_libdir}
-%{gem_instdir}/init.rb
-%{gem_instdir}/rails
+%exclude %{gem_instdir}/init.rb
+%exclude %{gem_instdir}/rails
```

Comment:
I've submitted https://github.com/rubiety/validates_lengths_from_database/pull/37 to drop these.

Reply:
Are these on the list to add to ignores in the templates?

Reply:
No, I manually modified it for now. Instead my hope is that by the time there's a new release my PR has been merged and there's no need to keep it in.

Reply:
Doesn't hurt though to add to the list? It took nearly a year for my inclusion of a LICENSE file in sprockets to get released :smile:

Reply:
It can. `rails` and `init.rb` could be included in some other package. And I don't expect us to regenerate other than when it's bumped.